src/Blog_App_frontend/src/AddBlog.jsx

```diff
@@ -0,0 +1,55 @@
+// AddBlog.jsx
+
+import React from 'react';
+import { Formik, Form, Field, ErrorMessage } from 'formik';
+import * as Yup from 'yup';
+import { useNavigate } from 'react-router-dom'; // Import useNavigate hook
+import './AddBlog.css'; // Import custom CSS file for styling
+
+// Define the validation schema using Yup
+const AddBlogSchema = Yup.object().shape({
+  title: Yup.string()
+    .max(125, 'Title must not exceed 125 characters')
+    .required('Title is required'),
+  description: Yup.string().required('Description is required'),
+});
```

Comment:
move to a constants file

Reply:
Change `AddBlogSchema` to `addBlogSchema`

Reply:
changed to addBlogs and moved the constants to a file constant.jsx